site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+
+**3:** Materialize button. When you click this button, you're directed to the **Create materializations** page.
+All the collections of the selected capture(s) will be added to the materialization.
+
+**4:** Filter captures by name.
+
+**5:** Shard status indicator. Shows the status of the task [shard](./advanced/shards.md) that backs this capture.
+
+* **Primary (Green)**: Data is actively flowing through the capture.
+* **Pending (Yellow)**: The capture is attempting to re-connect. Often, you'll see this after you re-enable the capture as Flow backfills historical data.
+* **Failed (Red)**: The capture has failed with an unrecoverable error.
+* **Disabled (White)**: The capture is disabled.
+* **Unknown (Black)**: The web app is unable to determine shard status. Usually, this is due to a temporary connection error.
+
+**6:** Capture name. The full name is shown, including all [prefixes](./catalogs.md#namespace).
+
+**7:** Capture type. The icon shows the type of source system data is captured from.
+
+**8:** Associated collections. The **Writes to** column shows all the collections to which the capture writes data. For captures with a large number of collections, hover over this column and scroll to view the full list.
+
+**9:** Publish time. Hover over this value to see the exact time the capture was first published.
+
+**10:** Options. Choose to **View Details** or **Edit Specification**.
+
+### Detail view
+
+When you click **View Details** for a capture, the **Status** and **Specification** viewers are revealed.
+
+The **Status** window shows the full identifier of the shard that backs your capture. If there's an error, you can view its logs.
+
+In the **Specification** window, you can view the specification of the capture itself, as well as each collection to which it writes.
+Select a specification from the **Files** list to view the JSON.
+
+### Editing captures and collections
+
+When you click **Edit specification** for a capture, you're taken to the **Edit Capture** page.
+
+This page is similar to the [**Create Capture**](#creating-a-capture) page as it was filled out just before the capture was published.
+
+To edit a capture:
+
+1. Edit the connection to the source system, if desired. You can either update fields in the **Endpoint Configuration** section or manually update the JSON in the **Specification Editor**.
+
+:::caution
+You may have to re-authenticate with the source system. Be sure to have current credentials on hand before editing the endpoint configuration.
+:::
+
+2. Use the **Collection Selector** to add or remove collections from the capture, if desired.
+
+   To refresh your connection with the source and see an updated list of possible collections, click the **Rediscover All** button,
```

Comment:
I believe Dave would prefer for the _Refresh_ CTA (formerly known as _Rediscover All_) to be renamed as _Find New_. I am not sure where he and Johnny stand on that revised name. Currently, _Refresh_ is the label associated with this CTA.